src/Test/Utilities/Desktop/ProcessUtilities.cs

```diff
@@ -78,6 +78,8 @@ public static class ProcessUtilities
 
                 process.WaitForExit();
 
+                Debug.Assert(process.HasExited);
```

Comment:
This means we're distrusting .WaitForExit? You might want a comment here. :smile: